prefixdb_batch.go

```diff
@@ -49,3 +49,8 @@ func (pb prefixDBBatch) WriteSync() error {
 func (pb prefixDBBatch) Close() error {
 	return pb.source.Close()
 }
+
+func (prefixDBBatch) Compact(start, end []byte) error {
+	// Not implemented
```

Comment:
can't we do the same as in [prefixdb.go Line 207](https://github.com/cometbft/cometbft-db/pull/111/files#diff-18ddb158554176aacb23a7b06bce0462dbe6df988813ccba23ccdcc362317948R207)? 

`	return pdb.db.Compact(start, end)
`

Reply:
prefixDBBatch does not have a db associated to it https://github.com/cometbft/cometbft-db/blob/3ae37a7fd546cd226bc82746f3fb1110fef97211/prefixdb_batch.go#L3-L6